### ISSUE TITLE
[UX/UI] create wireframes for mobile app

### ISSUE THREAD

**Issue by NathyNic** [unmsmfisi-socialapplication/social_app#22]
provide wireframe o mockups

**Comment by eds000n** [comment]
too many for this task, split it by components, there's also a web design that is needed

**Comment by NathyNic** [comment]
Yes, we have already planned to split this task, it's just that I don't know if we can make subtasks here linked to this one. So for now, we decided to assign everyone this task and make another one for each of us. Then, we planned to comment here with the link to our own task. hope you can understand that professor. Thanks :D
